### PR TITLE
new poison feature added

### PR DESCRIPTION
**Pull request by scohen40**

### CODE REVIEW

File: Snake/Snake.java [hunk 1]
```diff
@@ -145,6 +145,24 @@ void grow () {
     public void grow (int x) {
         size += x;
     }
+    
+    /**
+     * Cuts the snake by one tile.
+     */
+    void cutDown () {
+		size--;
+		if(size > 0) {
```
Comment (andrewoid) [inline Snake/Snake.java:154]: If your size is zero you should probably lose the game. This also means that the size variable will be out of sync with the length of the List.

File: Snake/GameBoard.java [hunk 2]
```diff
@@ -1,253 +1,311 @@
...
-
-                sb.append(" ");
-
-            }
-            sb.append("\n");
-        }
-
-        return new String(sb);
-    }
+class GameBoard extends JFrame {
```
Comment (andrewoid) [inline Snake/GameBoard.java:22]: According to the pull request it appears you changed every line of GameBoard. Please revert your changes and only modify the lines that you need to change.

File: Snake/Snake.java [hunk 3]
```diff
@@ -145,6 +145,24 @@ void grow () {
...
+			snakeList.remove(snakeList.size() - 1);
+		}
+    }
+    
+    /**
+     * Cuts the snake a specified number of tiles.
+     * @param x		the number of tiles to cut the snake by
+     */
+    void cutDown (int x) {
```
Comment (andrewoid) [inline Snake/Snake.java:163]: Is this method ever used?

**Review by andrewoid — CHANGES_REQUESTED** [review]

**Comment by scohen40** [comment]
How do you revert the changes?

**Pull request closed without merging**